test/unit/spec/v2/view-builder/utils/AuthenticatorUtil_spec.js

```diff
@@ -33,7 +13,7 @@ describe('v2/utils/AuthenticatorUtil', function() {
       ],
       'errorSummary': ''
     };
-    const result = removeRequirementsFromError(errorJSON, policy);
+    const result = removeRequirementsFromError(errorJSON);
```

Comment:
Is there a reason we dont have testcafe tests for this flow ?

Reply:
I am not sure, I believe since it was a util function, only unit tests were added.